README.md

```diff
@@ -1,3 +1,16 @@
 # smallrepo
 
 foo
+bar
+baz
+xpto
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc lacinia nibh et rhoncus ornare. Aliquam vel dapibus metus, vel tempus sem. Nunc consequat cursus purus id aliquet. Nulla facilisis metus eget elit finibus interdum. Mauris dapibus fermentum erat, non mattis enim egestas sed. Praesent mattis turpis ut arcu porttitor feugiat. Aenean pulvinar nunc nec consequat mattis.
+
+Fusce ac augue blandit elit faucibus malesuada. In nec purus ut est porttitor pellentesque. Suspendisse potenti. Nunc maximus placerat enim, vitae dictum dolor. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Pellentesque nec tempus est. Maecenas porta erat eget neque ullamcorper, ac malesuada nibh fermentum. Cras at rhoncus orci, in laoreet neque. In ac dolor arcu. Ut eu neque at sem egestas sollicitudin quis ac lectus.
```

Comment:
Multiline suggestion (lines 8-10)
```suggestion
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc lacinia nibh et rhoncus ornare. Aliquam vel dapibus metus, vel tempus sem. Nunc consequat cursus purus id aliquet. Nulla facilisis metus eget elit finibus interdum. Mauris dapibus fermentum erat, non mattis enim egestas sed. Praesent mattis turpis ut arcu porttitor feugiat. Aenean pulvinar nunc nec consequat mattis.

Fusce ac augue blandit elit faucibus malesuada. In nec purus ut est porttitor pellentesque. Suspendisse potenti. Nunc maximus placerat enim, vitae dictum dolor. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Pellentesque nec tempus est. Maecenas porta erat eget neque ullamcorper, ac malesuada nibh fermentum. Cras at rhoncus orci, in laoreet neque. In ac dolor arcu. Ut eu neque at sem egestas sollicitudin quis ac lectus.
```

Reply:
Reply to https://github.com/anotherkassio/smallrepo/pull/2#discussion_r718627265

Reply:
Reply to https://github.com/anotherkassio/smallrepo/pull/2#discussion_r741317101